### PR TITLE
PID Yaw LPF to 100Hz

### PR DESCRIPTION
**Pull request by JulioCesarMatias**
How about we try this?

https://github.com/betaflight/betaflight/pull/10841

### CODE REVIEW

File: docs/Settings.md [hunk 1]
```diff
@@ -5704,11 +5704,11 @@ These are values (in us) by how much RC input can be different before it's consi
 
 ### yaw_lpf_hz
 
-Yaw low pass filter cutoff frequency. Should be disabled (set to `0`) on small multirotors (7 inches and below)
+Yaw low pass filter cutoff frequency. Should be disabled (set to `0`)
```
Comment (avsaase) [inline docs/Settings.md]: If it should be disabled then the default should be 0.

**Comment by DzikuVx** [comment]
Actually, I'm not sure this is really needed. 100Hz is very close to 110Hz of gyro filter default. I mean, I would like to have some evidence that it does something useful before. None of my multiroptors benefited ever from any yaw LPF

**Comment by JulioCesarMatias** [comment]
> Actually, I'm not sure this is really needed. 100Hz is very close to 110Hz of gyro filter default. I mean, I would like to have some evidence that it does something useful before. None of my multiroptors benefited ever from any yaw LPF


I understood Pawel. But how about then just edit the description of the param?

Actual:
`Yaw low pass filter cutoff frequency. Should be disabled (set to `0`) on small multirotors (7 inches and below)`

New:
`Yaw P term low pass filter cutoff frequency. Should be disabled (set to `0`) on small multirotors (7 inches and below)`

**Comment by DzikuVx** [comment]
@JulioCesarMatias yes, let's update the description!

**Comment by JulioCesarMatias** [comment]
ok, i set the value back to zero, and edited the param description

**Pull request merged**